src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -59,6 +60,16 @@
 #endif
 #include "proto/capabilities.pb.h"
 
+/**
+ * @def OTBR_CONFIG_MESHCOP_EPHEMERAL_BORDER_AGENT_UDP_PORT
+ *
+ * Specifies the border agent UDP port for ephemeral meshcop service, and use 0 for ephemeral port.
```

Comment:
This documentation may cause confusion that the first 'ephemeral' is for the ephemeral key service and the second 'ephemeral' is for the ephemeral port of the OS.

Reply:
Modified accordingly, thanks for double checking.